### DISCUSSION TITLE
Please add the definition of other networks

### DISCUSSION THREAD

**Discussion by Gitcqs** [1200wd/bitcoinlib#307] [Ideas]
please at least support ETH and USDT

**Comment by Gitcqs** [comment]
At present, bitcoin and litecoin are very convenient to try out in the production environment (but ETH and USDT are implemented with other libraries), thank you in advance, maybe later versions add ETH and USDT👍👍👍

**Comment by yupasik** [comment]
Is there any blocking reason not to merge BCH & BCHSV [networks](https://github.com/1200wd/bitcoinlib/blob/add-bitcoin-cash/bitcoinlib/data/networks.json) definitions in release?
Thanks

**Comment by mccwdev** [comment]
I don't think those networks have enough stable and good working Blockchain service providers. Without a good API backend the library doesn't work as expected.
